File: ios/Classes/models/NotDisturbInfo.swift


public class NotDisturbInfo {
    /// 会话ID
    var conversationId: String?
    
    /// 免打扰状态
    /// 1: 屏蔽消息; 2: 接收消息但不提示; 0: 正常
    var result = 0
}

File: android/src/main/kotlin/io/openim/flutter_openim_sdk_ffi/models/NotDisturbInfo.kt
package io.openim.flutter_openim_sdk_ffi.models

class NotDisturbInfo {
    /**
     * 会话id
     */
    var conversationId: String? = null

    /**
     * 免打扰状态
     * 1:屏蔽消息; 2:接收消息但不提示; 0:正常
     */
    var result = 0
}